app/javascript/packs/banner_editor.jsx

```diff
@@ -180,19 +196,26 @@ export default class BannerEditor extends React.Component {
   async fetchBanner(){
 
     let that = this
+    let url;
+
+    if(this.state.defaultSiteBannerMode){
+      url = '/publishers/' + document.getElementById("publisher_id").value + "/site_banners/" + this.props.defaultSiteBanner.id + "?default=" + this.state.defaultSiteBannerMode
```

Comment:
Does it matter if the default param is passed in? 
If the site banner belongs to the current publisher then we might not even need to pass the param 🎉